OracleRAC/Vagrantfile

```diff
@@ -595,6 +595,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  # If var_asmlib_type is 'ASMFD', then the running kernel must be UEK4, otherwise ASMFD is not supported due to Grid Infrastructure being version 19.3
+  # It is worth noting that some GI RU later than 19.3 makes UEK5 supported as well, but not UEK6.
+   if var_asm_lib_type == 'ASMFD'
+     config.vm.provision :shell do |shell|
+       shell.privileged = true
+       shell.inline = <<-SHELL
+       echo "Installing UEK4 kernel (via yum)"
+       sed -i '/enabled=0/c\\enabled=1' /etc/yum.repos.d/uek-ol7.repo; yum install -y kernel-uek-4.1.12 kernel-uek-devel-4.1.12
+       echo "Rebooting to make UEK4 the running kernel, a prerequisite for ASMFD"
+       SHELL
+       shell.reboot = true
+     end
```

Comment:
Looks like your changes are fine on virtualbox but failing on libvirt because after the reboot (due to uek4) "/vagrant" is not mounted and then the deploy is failing. It's required to add following lines into Vagrantfile: (at line 610):
if var_provider_name == 'libvirt'
config.vm.provision :reload
end